examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
JTnE4HHpARLPOfhdatE3S8yig7DO5X5xvBqKgCKgCCgCioAzAkoMOaPhoedMOEybNg3Fh+3xyvXQbtbbZin7Jn78gaRRewQGDRokPWrhK1/7Hm+uLcePH39zDeg6jIY2CryWmWS9nsGk8P/982354F8zzY84JtRumzbFJNaYHKdygl7jS5aukN+9/kd574N/yy9//mMkU3u5sdsNsG3V04mZxNuGGdGzPv1MYpHYGg0br1FIDDh7u7uxA7qpIlAlAkwyd+zQXrbiHPzHm2/KABSxJ2FjBdUnEeER0hqF3p1VKNb7tfnLRAY/20eMGIEaKF/IzJkfGlKIxMlWzOCNjo6RB+6/D8Xt29umsuB+sv4NLetSUdi7P8bZHLOwOQPaSuaTDKLi5euvv0ayvZGxzpqAzwBa7d0owfsGE38k+lhraMTw4eWOW3FxsbErmg9LN5JevXr2tGVoxLA16jc9/tgjICwGoI/9UH2dNrPLWfg7BseUtaxIDvH42x0kuVgziYqvRx99uFYJ05ruE5OBF1G7iUlRi4ix2uCkElr38f7O885KPFrvu/PXmuxDApP2TmFhYaYmD23l9uD8JsZTp06RTlBwUb1k13Xrzj67sy3PLf4m6NGju1FAsU4YiSHes6go2rlzp1FV3EjEUAEUhbt375av5y80n/9WbS4Lp8a4L3ft2lUmTJ4ovfv0dosgt9qs6i9tQ6lsYF1dWprZSYhX1a++dzUCCQlthaTP2nUbzCQCqlGs4PVMq0gqSaiGrW0MHTII9+QokELrQKBfsTajW0ov/Gbs2bO743PRuY/evXoalekBkA1W8Pvz4MEDDcmyFaRRET5jqooh6DsdSv3klMOO1dgGbfOocKxpDEANNJ6vK1auNsodbu+D+29nKGkGDexf4TisPqgaIjHECAsLRc6mfM1Ba73K/lKtcxTfJ6igYpCg4mdBND6Hhg0bAgXWBjNZgu81RF6I9nX9+/cFifSVnIIquqKgYtoKkj+uNnnsK/uyTR7Xc66lZG3n3AY/e7hfSgxZ6OhfRUARUAQUAQsBJYYsJDz4lz/cONtNo3YI8Me04lc77Jy3iomJcX6pz2uIgDXTvIab6epOCNSHa5kqge+8/H0U6d5hyPofvfo9Yd0HJvRc46XvvmBsKd5+53359vMvoS7Fs/IgktS1CeY/8ZvsmjEGhbzfTEqWI5hZ/s6bb8m7b70Di6AeMmbcGBmEuhnEUEMR8AQCtHRhQnDVqtVy5uxZWb5sRVki+fKlwaT+6NGj5aEH7jfKC7v2gZNnmJT51reeMXVg0tLSoOYpQEIsRuLiYo3Swe6ENmfopmeky9x58+U/sPiiZQuTzNOn32UIYBa8X7lypfg1biId8P1t/fr1EhwcjITVKI+QGXZhabVDwiU4OEj69esHC7t98uc//83UaenerZshwfJQaPsAZjFv3LzZ1HN6+qknjMLG2t6Ov7TQYZ3OXkggUpnEGek81haZYUcfFbVBC6GQ0OZGUULCpC4mFZGwILF60cyA3ymdkCQMCixLLOYi2boXdkAnoCgaCQLUTrUOj+O27dtxjPdjzCHm82ENFAaLYWlHxevhlCMyd+58KGjiDXFUEV71fRkJTCY8k/C5SPulgwcPGUL3LO5RVFI0RS2jGKihHnrwAZMk5/l2IwSTtCSmZ34wU1avWgXCpyGuwbbSA/tPgo/nCesq7YcyageO8Xbcm3vivceefFzi4uM8NkRen0eOHpWf/eLXUFm0wD1/hEnUt4cayxNErscGcpM0zO98JEoYJA1PQwHi18jP3N+tiQyuQyVJ/NyzT7survR1XFys+aylmvPM2XPG+tPZOq6iDXl+jsHEpUEggs6CbODkV+faQ089+VhFm5Vb1ggkzoTxY8vGBXLEtY1yK1fyoqKxPgpCnEQK7x2hIHlIol0rigqLHKu0S2jreF7dJ5xg9vBDD5h7VUPYiVr13bg9SSO2yX3y8fE2k72sa+nee6ZX2gVtWVn7id9XaJnperxZ14mTZmjfx3Y7QPHtGnFxsTieTU2tKiqUWFtQQxFQBBQBRUARcEVAiSFXRPS1IqAIKAKKwE2LwG/+9/cOUujvf30DdhGV+4g3g93CE5j13hfWSz/40Y/lz3/5u5kJORQEjadixKiR8vabb0tEZITcAduIpYuXIiG0zTyYHCc5RCURE0SuPxI9tU/a7q2BAGe3Tpw4Af78FSvjmDAMRPKHNQ7sDiaZInHOs55jP8z6ZdKUy6zkid39sb3CwkKTeCVx0BPkK5O07777HsiwRmYmdCMk3ybDwo7qhGUgyagw6oMZ+5zRXNNggooKjr/89W+GbGPRaZLTrOvz+h/+6GiOtSM6oAYN6/K4G8SPs5Kff/7bUGN9aR6zoMrifYMWPryfDB48SB7DPa4bFAmeImx4DEkI8VEXQaXDmNFjYAH4Je7Zf5NpUMxERkY6uuY4SVrRdsiuMbMd1v/o1LGjzPzwI1kHlVaP7t2NFRIVLUdgHzRhwjhYNbaz9b5diGTmOcwY5znL8zIt7Rhm/q+VQYP6Q7H1GMjNVbIJShrWDHFO2jrAqOdPSCYSvzf+9BdTp4sTOKgE69q1i0yePMn8bYFjy3OZSgGe83YdU09Cw5n/u1FP5e9//bsch+1hd9xjHnnsUemI5K/rPY+k6lGcP1/O+gJk/XJj+/jI41DjDfbM9xAmtLtC0dcY59Sx48eMsvrjTz4DaRUvE5HEHz5sKI5BC0/Co21XggDPcypQPBUkt/moSZj6OtG1m7DE64BhxgVixc5wJmau1S7vM5u3bDOr8TOYyunahB9sLys7Prw/8TtOTYL2do888qBQVciJHhXFiOFDQYh3gXVskwrVhPzcZV09KoeohHK9v1TUZnWWsR1+TyN2nrznsu4Tw679rs7YdB1FQBFQBG5FBJQYuhWPuo5ZEVAEFIFbEIH5C76WhV8vMrMSr0UKOcPDH11/+P1v5Ymnvi0//dmv5KOZ79X4B55ze1U9
```

Comment:
I would add a bit more here on renaming & visibility for the published boards:
* rename the board and you can put it any project
* Board url visibility will match the project—so put it in a public project if you want to share with others
* by default the project will be "weave"